packages/flutter_tools/templates/create/pubspec.yaml.tmpl

```diff
@@ -26,7 +30,7 @@ flutter:
 
   # The following line ensures that the Material Icons font is
   # included with your application, so that you can use the icons in
-  # the Icons class.
+  # the material Icons class.
```

Comment:
please make sure there's a bug on file about moving us to be consistent for both material and cupertino

Reply:
will do https://github.com/flutter/flutter/issues/12569